### PR TITLE
Adds screenshot tests to OAuth PKCE

### PR DESCRIPTION
**Pull request by kul3r4**
- Adds screenshot tests to OAuth PKCE
- Adds previews and some refactoring for enable testing on screens
- clean up of library versions

### CODE REVIEW

File: WearOAuth/oauth-pkce/src/test/snapshots/images/com.example.android.wearable.oauth.pkce_AuthenticateScreenTest_authenticateFailedScreenTest[0].png
Comment (yschimke) [inline WearOAuth/oauth-pkce/src/test/snapshots/images/com.example.android.wearable.oauth.pkce_AuthenticateScreenTest_authenticateFailedScreenTest[0].png:1]: I guess string should be "Authentication failed"?

File: WearOAuth/oauth-pkce/src/test/snapshots/images/com.example.android.wearable.oauth.pkce_AuthenticateScreenTest_authenticateScreenTest[0].png
Comment (yschimke) [inline WearOAuth/oauth-pkce/src/test/snapshots/images/com.example.android.wearable.oauth.pkce_AuthenticateScreenTest_authenticateScreenTest[0].png:1]: What about a "\n" before "Name: ...."  So it's not squished?

**Review by yschimke — APPROVED** [review]

File: WearOAuth/oauth-pkce/src/test/snapshots/images/com.example.android.wearable.oauth.pkce_AuthenticateScreenTest_authenticateFailedScreenTest[0].png
Comment (kul3r4) [reply WearOAuth/oauth-pkce/src/test/snapshots/images/com.example.android.wearable.oauth.pkce_AuthenticateScreenTest_authenticateFailedScreenTest[0].png:1]: thanks!

File: WearOAuth/oauth-pkce/src/test/snapshots/images/com.example.android.wearable.oauth.pkce_AuthenticateScreenTest_authenticateScreenTest[0].png
Comment (kul3r4) [reply WearOAuth/oauth-pkce/src/test/snapshots/images/com.example.android.wearable.oauth.pkce_AuthenticateScreenTest_authenticateScreenTest[0].png:1]: thanks

**Pull request merged**